dbms/tests/queries/0_stateless/00486_pk_subexpression.reference

```diff
@@ -0,0 +1,20 @@
+2000-01-01      1970-01-01 00:00:01     11      1235
+2000-01-01      1970-01-01 00:00:02     11      4395
+2000-01-01      1970-01-01 00:00:03     22      3545
+2000-01-01      1970-01-01 00:00:04     22      6984
+2000-01-01      1970-01-01 00:00:05     33      4596
+2000-01-01      1970-01-01 00:02:03     33      1235
+2000-01-01      1970-01-01 00:02:01     33      2791
+2000-01-01      1970-01-01 00:02:02     33      2791
+2000-01-01      1970-01-01 00:02:05     44      4578
+2000-01-01      1970-01-01 00:02:04     44      4935
+2000-01-01      1970-01-01 00:02:08     55      1235
+2000-01-01      1970-01-01 00:02:07     55      2791
+2000-01-01      1970-01-01 00:02:06     55      5786
+2000-01-01      1970-01-01 00:00:01     11      1235
+2000-01-01      1970-01-01 00:01:03     11      3572
+2000-01-01      1970-01-01 00:01:01     11      4563
+2000-01-01      1970-01-01 00:01:02     11      4578
+2000-01-01      1970-01-01 00:01:03     11      3572
+2000-01-01      1970-01-01 00:01:01     11      4563
+2000-01-01      1970-01-01 00:01:02     11      4578
```

Comment:
No newline at end of file, and this differs from query result, that always have newline.